database/config/config.json

```diff
@@ -1,17 +1,19 @@
 {
   "development": {
-    "username": "root",
-    "password": null,
-    "database": "database_test",
-    "host": "127.0.0.1",
-    "dialect": "postgres"
+    "username": "zdhxckgo",
+    "password": "WkcWr_Wr_mlocngTdceWq_NWOo2XPvoO",
+    "database": "zdhxckgo",
+    "host": "elmer.db.elephantsql.com",
+    "port": "5432",
+    "dialect": "postgres",
+    "logging": false
   },
```

Comment:
Should these credentials be committed?

Reply:
My thought was that this was just going to be a dummy account so we wouldn’t need to bother not pushing the config stuff to make things simpler, but after talking to Daniel he made some good points as to why we shouldn’t do that so I’ll probably return everything to it’s basic configuration before merging to master.

I’ll then change the password and then update our google doc with the new correct login info.

Reply:
Gotcha; dang. For a fleeting moment I thought I was going to get out of swapping that back in forth in the future. :-P